app/controllers/Contributions.scala

```diff
@@ -99,7 +99,7 @@ class Contributions(paymentServices: PaymentServices, addToken: CSRFAddToken, cl
         CSRF.getToken.map(_.value),
         request.isAllocated(Test.landingPageTest, "with-copy"),
         disableStripe
-      ))
+      )).addingToSession("payment_session" -> request.sessionId)
```

Comment:
What does adding the `payment_session` value to the session give us? As opposed to just using `initialSessionId`?

Reply:
I think something like `contributions_session` would be a more informative name.

Reply:
changed all references to "payment_session" to read "contributions_session" in next commit